app/views/users/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>New user</h1>
+
+<%= render "form", user: @user %>
+
+<br>
+
+<div>
+  <%= link_to "Back to users", users_path %>
```

Comment:
- [ ] it would be better if you match the wireframe as required in order to complete the requiremenets 
- wireframe 
![blog_root_users](https://user-images.githubusercontent.com/61790587/208241777-eb4278da-3c3a-43fe-8b1a-8c27b6f71fb4.png)
- Yours
<img width="415" alt="Screenshot 2022-12-17 at 13 17 10" src="https://user-images.githubusercontent.com/61790587/208241782-1ccc3f8b-99a9-4f19-b077-e069b4870900.png">